### PR TITLE
[zh-cn] sync some en docs

### PR DESCRIPTION
**Pull request by my-git9**
```
#       modified:   content/zh-cn/docs/concepts/overview/working-with-objects/owners-dependents.md
#       modified:   content/zh-cn/docs/concepts/windows/intro.md
#       modified:   content/zh-cn/docs/concepts/windows/user-guide.md
#       modified:   content/zh-cn/docs/reference/kubernetes-api/workload-resources/controller-revision-v1.md
#       modified:   content/zh-cn/docs/reference/kubernetes-api/workload-resources/replication-controller-v1.md
#       modified:   content/zh-cn/docs/tutorials/hello-minikube.md
#       modified:   content/zh-cn/docs/tutorials/security/cluster-level-pss.md
#       modified:   content/zh-cn/docs/tutorials/services/connect-applications-service.md
```

### CODE REVIEW

**Comment by howieyuen** [comment]
CI jobs are failed, make a force push to trigger it again

File: content/zh-cn/docs/concepts/overview/working-with-objects/owners-dependents.md [hunk 1]
```diff
@@ -11,37 +11,37 @@ weight: 60
...
 an owner reference. Owner references help different parts of Kubernetes avoid
 interfering with objects they don’t control. 
 -->
 属主关系不同于一些资源使用的[标签和选择算符](/zh-cn/docs/concepts/overview/working-with-objects/labels/)机制。
 例如，有一个创建 `EndpointSlice` 对象的 Service，
-该 Service 使用标签来让控制平面确定，哪些 `EndpointSlice` 对象属于该 Service。
-除开标签，每个代表 Service 所管理的 `EndpointSlice` 都有一个属主引用。
-属主引用避免 Kubernetes 的不同部分干扰到不受它们控制的对象。
+该 Service 使用{{<glossary_tooltip text="标签" term_id="label">}}来让控制平面确定，
+哪些 `EndpointSlice` 对象属于该 Service。除开标签，每个代表 Service 所管理的
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/owners-dependents.md:35]:
```suggestion
该 Service 使用{{<glossary_tooltip text="标签" term_id="label">}}来让控制平面确定哪些 
`EndpointSlice` 对象属于该 Service。除开标签，每个代表 Service 所管理的
```

**Review by tengqm — COMMENTED** [review]
/approve

**Review by kinzhi — COMMENTED** [review]
/lgtm

**Pull request merged**